config.json

```diff
@@ -28,6 +29,11 @@
   ],
   "exercises": [
     {
+      "slug": "hello-world",
+      "difficulty": 1,
+      "topics": []
```

Comment:
All the difficulties are set to 1, which is the default. They are in the process of being set correctly, but it takes some time